### PR TITLE
Using buffered writer and not using go routines for now

### PR DESCRIPTION
**Pull request by aswinkarthik93**

### CODE REVIEW

File: characters.go [hunk 1]
```diff
@@ -2,9 +2,11 @@ package main
 
 import "math/rand"
 
-const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!@#$%^&*()-=_+,./;'[]<>?:{}"
 const total = 89
 
+var allCharacters = []string{"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s", "t", "u", "v", "w", "x", "y", "z", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "!", "@", "#", "$", "%", "^", "&", "*", "(", ")", "-", "=", "_", "+", ",", ".", "/", ";", "'", "[", "]", "<", ">", "?", ":", "{", "}"}
```
Comment (kishaningithub) [inline characters.go]: Was the "String" version causing performance issues ?

Comment (kishaningithub) [inline characters.go]: Was the "String" version causing performance issues ?

**Comment by aswinkarthik93** [comment]
Seeing a 7MBps dip in write speed. could you check it on ur machine too?

**Comment by aswinkarthik93** [comment]
Yes. Found 7MBps dip when we use a single string instead of string array

**Pull request merged**